src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -32,6 +32,53 @@
 FRONTEND_ASIC_SUB_ROLE = "FrontEnd"
 BACKEND_ASIC_SUB_ROLE = "BackEnd"
 
+# Chassis STATE_DB keys
+CHASSIS_INFO_TABLE = 'CHASSIS_INFO|chassis {}'
+CHASSIS_INFO_CARD_NUM_FIELD = 'module_num'
+CHASSIS_INFO_SERIAL_FIELD = 'serial'
+CHASSIS_INFO_MODEL_FIELD = 'model'
+CHASSIS_INFO_REV_FIELD = 'revision'
+
```

Comment:
Per PEP8, there should be two blank lines before function definitions. Also suggest moving the new functions to the end of the section, mainly because `get_platform_info` relies on other functions. Even though Python isn't picky about definition order, I still prefer it. Suggest moving these functions directly above

```python
#
# Multi-NPU functionality
#
```

Reply:
Are there any linting tools built into the SONiC codebase as of now?

If not, I would highly suggest adding some linting tools to the push gate checks or adding a auto-formatter / checker to the repo if style guidelines are going to be enforced. PEP8 is a massive standard and knowing precisely what we want to standardize on will help developers push code that they know matches our guidelines.

Reply:
Currently we don't have linting tools built in. We're looking into it as a low priority. In the meantime, I suggest using the `autopep8` tool. Fee free to adjust the max line length to ~100-120 (e.g., `autopep8 --max-line-length 120 <file>`)